src/app/process-editor/services/cardview-properties/called-element-item/called-element-item.component.html

```diff
@@ -0,0 +1,28 @@
+<div class="called-element-block">
+    <mat-form-field>
+        <div class="adf-property-label">{{ 'PROCESS_EDITOR.ELEMENT_PROPERTIES.ACTIVITY_NAME' | translate }}</div>
+        <mat-select (selectionChange)="changeProcess()" [(ngModel)]="processId">
+            <mat-option *ngFor="let process of processes$ | async" [value]="process.id">
```

Comment:
Can you please remove the current process from the list? I don't think it makes any sense to refer to the very same process. Does it?